gnomad/utils/transcript_annotation.py

```diff
@@ -71,3 +71,77 @@ def summarize_rsem_mt(
         )
 
     return transcript_ht.key_by("transcript_id", "gene_id"), gene_ht.key_by("gene_id")
+
+
+def get_expression_proportion(
+    transcript_ht: hl.Table,
+    gene_ht: hl.Table,
+    tissues_to_filter: Optional[List[str]] = None,
+    tissue_as_row: bool = False,
+) -> hl.Table:
+    """
+    Calculate the proportion of expression of transcript to gene per tissue.
+
+    :param transcript_ht: Table of summarized transcript expression by tissue
+    :param gene_ht: Table of summarized gene expression by tissue
+    :param tissues_to_filter: Optional list of tissues to filter out
+    :param tissue_as_row: If True, the input Table is with a row annotation for each
+        tissue instead of an array of values. Default is False.
+    :return: Table with expression proportion of transcript to gene per tissue
+        and mean expression proportion across tissues
+    """
+    if tissue_as_row:
+        tissues1 = list(gene_ht.row)
+        tissues1.remove("gene_id")
+        gene_ht = gene_ht.select(
+            gene_expression=hl.array([gene_ht[tissue] for tissue in tissues1])
+        ).annotate_globals(tissues=tissues1)
+
+        tissues2 = list(transcript_ht.row)
+        tissues2.remove("transcript_id")
+        tissues2.remove("gene_id")
+        transcript_ht = transcript_ht.select(
+            transcript_expression=hl.array(
+                [transcript_ht[tissue] for tissue in tissues2]
+            )
+        ).annotate_globals(tissues=tissues2)
```

Comment:
Now this can just use the function I put into my PR

```suggestion
    transcript_ht = tissue_expression_ht_to_array(
        transcript_ht, tissues_to_filter=tissues_to_filter
    )
    gene_ht = tissue_expression_ht_to_array(
        gene_ht, tissues=hl.eval(transcript_ht.tissues)
    )
```